lib/std/build/RunStep.zig

```diff
@@ -148,6 +166,42 @@ fn stdIoActionToBehavior(action: StdIoAction) std.ChildProcess.StdIo {
     };
 }
 
+fn getExternalExecutor(artifact: *LibExeObjStep) !?[]const u8 {
```

Comment:
Again, like in the comment above, why do we need to duplicate `getExternalExecutor` here?

Reply:
It doesn't duplicate the code but the name. I guess I can rename it to avoid confusion. This function utilizes CrossTarget's `getExternalExecutor()`, but besides that have a check if the executor is enabled. It's similar to the logic for [tests](https://github.com/ziglang/zig/blob/master/lib/std/build.zig#L2520).